baremaps-core/src/main/java/org/apache/baremaps/iploc/database/InetnumLocationDaoSqliteImpl.java

```diff
@@ -38,13 +38,12 @@ public final class InetnumLocationDaoSqliteImpl implements InetnumLocationDao {
   private static final String INSERT_SQL =
       "INSERT INTO inetnum_locations(address, ip_start, ip_end, latitude, longitude, network, country) VALUES(?,?,?,?,?,?,?)";
 
-  private static final String SELECT_ALL_SQL =
-      "SELECT " + "id, \n" + "address, \n" + "ip_start, \n" + "ip_end, \n" + "latitude, \n"
-          + "longitude, \n" + "network, \n" + "country \n" + " FROM inetnum_locations;";
+  private static final String SELECT_ALL_SQL = "SELECT id, address, ip_start, ip_end, latitude, "
```

Comment:
You can use the new multi-line string syntax for this kind of queries.

```java
String SELECT_ALL_SQL = """
    SELECT id, address, ip_start, ip_end, latitude,
    FROM inetnum_locations 
    WHERE ip_start <= ? AND ip_end >= ?;"""
```

Reply:
Thank you, I have updated to triple double quotes!